llama_stack/distribution/datatypes.py

```diff
@@ -117,6 +117,22 @@ class Provider(BaseModel):
     config: Dict[str, Any]
 
 
+class LogCategoryConfig(BaseModel):
+    category: str = Field(
+        default="server",
+        description="logging category to apply this configuration to (core | server | router | inference | agents | safety | eval | tools | client)",
```

Comment:
It seems from the below code that `all` is also a valid option?

Reply:
yup, added that in